test/sagemaker_tests/autogluon/__init__.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from __future__ import absolute_import
+import sagemaker
+
+def invoke_autogluon_helper_function(ecr_image, sagemaker_regions, helper_function, *helper_function_args):
+    """
+    Used to invoke SM job defined in the helper functions in respective test file. The ECR image and the sagemaker
+    session are passed explicitly depending on the AWS region.
+    This function will rerun for all SM regions after a defined wait time if capacity issues are seen.
+
+    :param ecr_image: ECR image in us-west-2 region
+    :param sagemaker_regions: List of SageMaker regions
+    :param helper_function: Function to invoke
+    :param helper_function_args: Helper function args
+
+    :return: None
+    """
+    from .. import get_ecr_image_region, get_sagemaker_session, get_ecr_image
+
```

Comment:
Can we add `num_retries` and `retry_delay` to this functionality as implemented [here](https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/integration/sagemaker/__init__.py#L52)

Reply:
Same with other `__init__.py` files

Reply:
As we discussed, retried tests likely fail because of timeout. So I didn't add it.